dockerfiles/launcher/launcher_funcs.sh

```diff
@@ -166,6 +166,22 @@ docker_run_with_debug() {
   fi
 }
 
+docker_run_with_volumes_lib() {
+  if has_lib_volume; then
+    docker_run_with_debug -v "${CHE_LIB_VOLUME}":/data/lib "$@"
```

Comment:
My pull request should now contain the required changes to [che.env](https://github.com/eclipse/che/pull/3252/commits/c3a6d78fc47f2f19d550224195842935fb485fc7#diff-e4ebb68a2356f42877c3d809f0e569ad).  I've left the previous changes to the launcher scripts intact for now, even though from my understanding the new CLI will make them redundant.